cypress/integration/userCanViewTheirCompletedRequests.feature.js

```diff
@@ -0,0 +1,69 @@
+describe("User can view their completed reQuests", () => {
+  beforeEach(() => {
+    cy.stubMain();
+    cy.StubRequestCompleted();
+    cy.login();
+    cy.get("#myrequest-home-link").click();
+    cy.wait(500);
+    cy.get("#requests-link").click();
+  });
+
+  it('in a list, by navigating to "completed reQuests"', () => {
+    cy.get("#completed-link").click();
+    cy.get("#request-7").within(() => {
+      cy.get(".header").should("contain", "Teach me Korean");
+    });
+    cy.get("#request-8").within(() => {
+      cy.get(".header").should("contain", "Im in prison");
+    });
+  });
+
+  it("can view a specific reQuest and its offer message", () => {
+    cy.get("#completed-link").click();
+    cy.get("#request-7").click();
+    cy.wait(1000);
+    cy.get("#request-description-7").should("be.visible");
+    cy.get("#offers > .cards > .ui > .content > .meta").should(
+      "contain",
+      "pauline@mail.com"
+    );
+    cy.get("#offer-message").should("contain", "Im a natural, i can teach you");
+  });
+
+  it("when a second reQuest is clicked, it's expanded instead of the first", () => {
+    cy.get("#completed-link").click();
+    cy.get("#request-8").click();
```

Comment:
should click on #request-7 first here, actually

Reply:
hm i believe this is the second one...